src/gadgets/r1cs.rs

```diff
@@ -237,7 +237,6 @@ impl<E: Engine> AllocatedRelaxedR1CSInstance<E> {
     // Compute r:
     let mut ro = E::ROCircuit::new(ro_consts, NUM_FE_FOR_RO);
     ro.absorb(params);
-    self.absorb_in_ro(cs.namespace(|| "absorb running instance"), &mut ro)?;
```

Comment:
Could we add a note here that we are not absorbing the running instance because "fill in"?